### ISSUE TITLE
Using work from boost-starai/boost-starai.github.io

### ISSUE THREAD

**Issue by batflyer** [testvon/testvon.github.io#45]
I am a little confused on the exact reason that our lab's [authors](https://github.com/testvon/testvon.github.io/commit/301923a961b2f878151dddac68b13fdb883d0594) and [photos](https://github.com/testvon/testvon.github.io/commit/746328fb6b5a384a27a69ec36980d95787948537#diff-30ed7664825e843152332d058c0d5a0f) are showing up in this repository, but since you are using them, you should really update your LICENSE.txt to reflect this.

[boost-starai/boost-starai.github.io](https://github.com/boost-starai/boost-starai.github.io) is under a [creative commons license](https://github.com/boost-starai/boost-starai.github.io/blob/master/LICENSE.txt). Using our work is fine so long as the conditions of the license are satisfied, in this case: providing credit and license notice.

**Comment by testvon** [comment]
Yes, I am sorry about this problem. I am using these data for testing some function on my page. I will update my LICENSE.txt soon. Thanks for your notice.

**Comment by batflyer** [comment]
I figured it might be something along those lines. If you're using them for testing and stop using them down the road, you can set the attribution now and update it later to reflect that they're no longer being used.

Are you doing a page for a mathematics club?
